editor/src/clj/editor/sprite.clj

```diff
@@ -271,7 +271,7 @@
   (or (validation/prop-error :fatal _node-id :material validation/prop-nil? material "Material")
       (validation/prop-error :fatal _node-id :material validation/prop-resource-not-exists? material "Material")))
 
-(g/defnk produce-build-targets [_node-id resource textures texture-binding-infos default-animation material material-attribute-infos material-max-page-count material-samplers material-shader blend-mode size-mode manual-size slice9 dep-build-targets offset playback-rate vertex-attribute-bytes vertex-attribute-overrides]
+(g/defnk produce-build-targets [_node-id resource textures texture-binding-infos default-animation material material-attribute-infos material-max-page-count material-samplers material-shader blend-mode size-mode manual-size slice9 offset playback-rate vertex-attribute-bytes vertex-attribute-overrides]
```

Comment:
In the large project I tested, most of the build-target -> dep-build-target connections in the graph were from sprites to their materials and from factories to their prototypes. Making these dependencies dynamic did not affect the memory usage, but it shaved off a minute from the total build time.